prog/weavedns/main.go

```diff
@@ -46,6 +48,7 @@ func main() {
 	flag.StringVar(&domain, "domain", weavedns.DefaultLocalDomain, "local domain (ie, 'weave.local.')")
 	flag.IntVar(&wait, "wait", 0, "number of seconds to wait for interface to be created and come up")
 	flag.IntVar(&dnsPort, "dnsport", weavedns.DefaultServerPort, "port to listen to DNS requests")
+	flag.StringVar(&httpIfaceName, "http-iface", "", "interface on which to listen for HTTP requests")
```

Comment:
No dashes, please. Yes, I know some have crept in, but for more exotic options only.

Should mention what the default is.